pages/docs/scale/kubernetes.html

```diff
@@ -48,12 +48,12 @@
         image: crate:latest
```

Comment:
to be honest the whole file doesn't really work anymore (k8s renamed petset to stateful set in 1.6, so it really works only in 1.5 with alpha API.... we need to rework this